src/handlers/swap_handlers.ts

```diff
@@ -224,7 +225,13 @@ export class SwapHandlers {
         );
         if (params.includePriceComparisons && quote.priceComparisonsReport) {
             const side = params.sellAmount ? MarketOperation.Sell : MarketOperation.Buy;
-            const priceComparisons = priceComparisonUtils.getPriceComparisonFromQuote(CHAIN_ID, side, quote);
+            const priceComparisons = priceComparisonUtils.getPriceComparisonFromQuote(
+                CHAIN_ID,
+                side,
+                quote,
+                this._swapService.slippageModelManager,
```

Comment:
Adding the `expectedSlippage` in the `priceComparisons`, mostly for quick insight/debugging.